cmd/grafanactl/resources/pull.go

```diff
@@ -118,11 +119,19 @@ func pullCmd(configOpts *cmdconfig.Options) *cobra.Command {
 				StopOnError: opts.StopOnError,
 			}
 
-			if err := writer.Write(ctx, res.Resources); err != nil {
+			// Strip server fields from the resources.
```

Comment:
Should we also do it for the `edit` command? Or not? :thinking:

Reply:
No, I think in `edit` we want to preserve all fields as is, because editing is one place where we _do_ need to ensure version consistency, e.g. if the object changes while it's being edited the changes shouldn't be applied.